doc/conf.py

```diff
@@ -84,7 +94,7 @@
     # execute files that match the following filename pattern,
     # and skip those that don't. If the following option is not provided,
     # all example scripts in the 'examples_dirs' folder will be skiped.
-    'filename_pattern': r'run',
+    'filename_pattern': r'/run_',
```

Comment:
Just our of curiosity: was this change needed even after renaming the tutorials?